### DISCUSSION TITLE
Suggest updating your README

### DISCUSSION THREAD

**Discussion by levbrie** [Kong/insomnia#3834] [Feedback]
The repo's description states:

"The Open Source API Client and Design Platform for GraphQL, REST and gRPC".

but the very first line of the README says only "Insomnia is a cross-platform REST client", making it unclear as to whether it does in fact support GraphQL and gRPC.

**Comment by dimitropoulos** [comment]
Thanks for this suggestion!  I've opened https://github.com/Kong/insomnia/pull/3985.

**Reply by wdawson** [reply]
This has been merged. Thanks @levbrie !
